packages/block-library/src/media-text/deprecated.js

```diff
@@ -366,4 +371,98 @@ export default [
 			);
 		},
 	},
+	{
```

Comment:
Do we actually need a new deprecation here if we are only adding a min-height style to the existing markup? The previous version's markup would still be valid correct?

Reply:
You're right. I was able to remove it.